intro.tex

```diff
@@ -1,55 +1,67 @@
 \section{Introduction}
 
-This  note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce early science during the time between commissioning and the first data release in year 1. 
-The current planned start date for full survey operations is 1 April 2024, with considerable uncertainty. 
-As of November 2021, the construction project schedule shows completion between December 2023 and July 2024 (i.e. planned finish and finish with contingency). 
-It is unlikely the project will finish in December 2023 and highly likely it will finish by July 2024. 
+This note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce high-impact early science during the time between commissioning and the first LSST Data Release, DR1.  
+Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
```

Comment:
```suggestion
Community expectations for early science are high due to the transformative nature of the Rubin data and the densely-sampled observations planned in commissioning and science validation.
```

not sure we can really call it "extensive time" especially given later discussion of the expected reductions

Reply:
True - this is out of date now.